.github/workflows/haskell-ci.yml

```diff
@@ -14,17 +14,17 @@ jobs:
         os: [ubuntu-latest]
         cabal: ["3.10.1.0"]
         ghc:
-          - 8.10.7
```

Comment:
Removing 8.10.7 not because it's broken but because most people should have migrated off of 8.x and not to overload CI unnecessarily.